src/APL/rest-apl.ts

```diff
@@ -30,14 +30,14 @@ export class RestAPL implements APL {
     this.headers = config.headers;
   }
 
```

Comment:
as agreed , lets rename to SaleorCloudAPL, hide implementation details here and drop generic REST APL idea